lib/muontrap/daemon.ex

```diff
@@ -23,23 +23,24 @@ defmodule MuonTrap.Daemon do
   is a list of options. The same options as `MuonTrap.cmd/3` are available with
   the following additions:
 
-  * `:name` - Name the Daemon GenServer
-  * `:log_output` - When set, send output from the command to the Logger. Specify the log level (e.g., `:debug`)
-  * `:log_prefix` - Prefix each log message with this string (defaults to the program's path)
+  * `:name` - Name the Daemon GenServer.
+  * `:msg_callback` - When set, it sends the output from the command to the callback. Only funtions with /1 arity are allowed.
+  * `:log_output` - When set, it sends the output from the command to the Logger. Specify the log level (e.g., `:debug`).
+  * `:log_prefix` - Prefix each log message with this string (defaults to the program's path).
   * `:stderr_to_stdout` - When set to `true`, redirect stderr to stdout. Defaults to `false`.
 
   If you want to run multiple `MuonTrap.Daemon`s under one supervisor, they'll
   all need unique IDs. Use `Supervisor.child_spec/2` like this:
 
   ```elixir
-  Supervisor.child_spec({MuonTrap.Daemon, ["my_server"), []]}, id: :server1)
+  Supervisor.child_spec({MuonTrap.Daemon, ["my_server", []]}, id: :server1)
```

Comment:
Could you separate this typo fix out into a separate PR? This fix is an easy one for me to merge right away.